conf/config-default.yaml

```diff
@@ -14,122 +14,121 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# PLEASE DO NOT UPDATE THIS FILE!
-# If you want to set the specified configuration value, you can set the new
-# value in the conf/config.yaml file.
+# CAUTION: DO NOT MODIFY DEFAULT CONFIGURATIONS IN THIS FILE.
+# Keep the custom configurations in conf/config.yaml.
 #
 
 apisix:
-  # node_listen: 9080               # APISIX listening port
-  node_listen:                      # This style support multiple ports
+  # node_listen: 9080          # APISIX listening port.
+  node_listen:                 # APISIX listening ports.
     - 9080
-  #   - port: 9081
-  #     enable_http2: true          # If not set, the default value is `false`.
-  #   - ip: 127.0.0.2               # Specific IP, If not set, the default value is `0.0.0.0`.
-  #     port: 9082
-  #     enable_http2: true
-  enable_admin: true
-  enable_dev_mode: false            # Sets nginx worker_processes to 1 if set to true
-  enable_reuseport: true            # Enable nginx SO_REUSEPORT switch if set to true.
-  show_upstream_status_in_response_header: false # when true all upstream status write to `X-APISIX-Upstream-Status` otherwise only 5xx code
+    # - port: 9081
+    #   enable_http2: true     # If not set, default to `false`.
+    # - ip: 127.0.0.2          # If not set, default to `0.0.0.0`/
+    #   port: 9082
+    #   enable_http2: true
+  enable_admin: true           # Admin API
+  enable_dev_mode: false       # If true, nginx worker_processes will be set to 1.
+  enable_reuseport: true       # If true, nginx SO_REUSEPORT option will be enabled.
```

Comment:
Is the passive voice better than the active voice, but Grammarly often tells me not to use it?

Reply:
Grammarly isn't right all the time.

Reply:
The most important feature of Grammarly is to let you change passive voice to active voice, i think it shoud make some sense.